accessibility_monitoring_platform/apps/notifications/templates/notifications/task_list.html

```diff
@@ -7,12 +7,6 @@
 
 {% block content %}
 <div class="govuk-width-container">
-    <div class="govuk-breadcrumbs">
```

Comment:
The figma diagram for this page sometimes still has these breadcrumbs which I assume is a mistake.